public/app/features/scenes/querying/SceneQueryRunner.ts

```diff
@@ -38,6 +42,11 @@ export interface DataQueryExtended extends DataQuery {
   [key: string]: any;
 }
 
+export interface TimeOverrideResult {
```

Comment:
Can we move this to @grafana/data? Or schema?

Reply:
I don't think it is needed, since this is just the type of data returned by the `applyPanelTimeOverrides` function. It is not used anywhere else